tests/test-recipes/metadata/cdt_linking/conda_build_config.yaml

```
@@ -0,0 +1,7 @@
+channel_sources:
+  - conda-forge
+channel_targets:
+  - conda-forge main
+
+cdt_name:          # [linux]
+  - cos7           # [linux]
```

For conda-forge, this should be
```suggestion
cdt_name:          # [linux]
  - conda          # [linux]
```
since https://github.com/conda-forge/cdt-builds/commit/dc270e432d9376389b3a36c918aba54cb0d28d5c